### PR TITLE
[UXE-5191] feat: add digital certificate drawer to domains page

### PR DESCRIPTION
**Pull request by aloisio-m-bastian**
### What is the new behavior introduced by this PR?
add digital certificate drawer to domains page
image

### Does this PR introduce breaking changes?
- [ ] No
- [ ] Yes 

### Does it have a link on Figma?

<hr />

#### Make sure your pull request fits the checklist below (when applicable):

- [x] The issue title follows the format: [ISSUE_CODE] TYPE: TITLE
- [x] Commits are tagged with the right word (feat, test, refactor, etc)
- [x] Application responsiveness was tested to different screen sizes
- [x] Code is formatted and linted
- [x] Tags are added to the PR

#### These changes were tested on the following browsers:
- [x] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] Safari

### CODE REVIEW

File: vite.config.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ import istanbul from 'vite-plugin-istanbul'
 
 const getConfig = () => {
   const env = loadEnv('development', process.cwd())
-  const URLStartPrefix = env.VITE_ENVIRONMENT === 'production' ? 'https://' : 'https://stage-'
+  const URLStartPrefix = env.VITE_ENVIRONMENT !== 'production' ? 'https://' : 'https://stage-'
```
Comment (HerbertJulio) [inline vite.config.js:11]:
```suggestion
  const URLStartPrefix = env.VITE_ENVIRONMENT !== 'production' ? 'https://' : 'https://stage-'
```

Comment (HerbertJulio) [reply vite.config.js:11]: Esse código não pode ser alterado, pois altera a forma que o ambiente local é levantado.

Comment (aloisio-m-bastian) [reply vite.config.js]: resolvido, havia esquecido de

**Review by vinigfer — APPROVED** [review]

File: src/views/DigitalCertificates/FormFields/FormFieldsCreateDigitalCertificates.vue [hunk 2]
```diff
@@ -8,7 +8,11 @@
   import { computed, watch } from 'vue'
 
   defineProps({
-    certificateSelection: String
+    certificateSelection: String,
+    isDrawer: {
+      type: Boolean,
+      default: false
```
Comment (pauloSF0) [inline src/views/DigitalCertificates/FormFields/FormFieldsCreateDigitalCertificates.vue:14]: o valor default já é falso

Comment (pauloSF0) [reply src/views/DigitalCertificates/FormFields/FormFieldsCreateDigitalCertificates.vue:14]:
```
   isDrawer:  {
      type: Boolean
   }
```

**Review by pauloSF0 — APPROVED** [review]

**Pull request merged**